independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/providers/serialisers/InputStreamMessageBodyHandler.java

```diff
@@ -26,9 +26,17 @@ public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotat
         return InputStream.class.isAssignableFrom(type);
     }
 
+    protected boolean doIsWriteable(Class<?> type) {
+        return InputStream.class.isAssignableFrom(type);
```

Comment:
I think we can name it `isInputStream(Class<?>)` since that seems to be the intention of this method :)

Reply:
Actually I think this method is not needed and you can call the one-liner directly